### DISCUSSION TITLE
Fundamentos do ASP.NET 6 - Enviando Email

### DISCUSSION THREAD

**Discussion by joifogaca** [balta-io/forum#465] [Tira Dúvidas] [answered]
Boa tarde, Estou na aula Enviando Email do curso de asp.net, segui todos os passos não deu nenhum erro, mas o e-mail tbm não chegou na minha caixa de entrada nem no spam, será que tem haver com os bloqueios da conta gratuita que foi citado no curso?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @joifogaca, como vai?

Sim, o SendGrid sofreu muito com Spam no ano passado e modificaram as políticas, tornando-as ainda mais restritas.
E-mails de domínios como Gmail, Hotmail e outros populares, até onde sei eles estão bloqueando bastante.

Uma alternativa é fazer o uso do SMTP Client nativo do .NET:
https://pt.stackoverflow.com/questions/630/como-posso-enviar-um-e-mail-pelo-gmail

Lembrando que é uma alternativa apenas para estudos! Como comento no curso, o envio de E-mails é algo mais complexo que recomendo fazer por alguma plataforma.
